diagnostics-app/src/app/rma/test-viewer/test-viewer.component.ts

```diff
@@ -58,6 +56,15 @@ export class TestViewerComponent {
     });
   }
 
+  ngAfterViewInit() {
```

Comment:
Added a comment to say why we need this `ngAfterViewInit`. For  reading dynamic variable in child component?

Reply:
Done.